src/services/analysisService.ts

```
@@ -43,20 +50,98 @@ export async function analyzeError(
+}
+
 export async function suggestCorrections(command: string): Promise<void> {
   const availableCommands = getAvailableCommands();
+
+  // Check if AI is enabled
+  if (!aiConfigService.isAIEnabled()) {
+    // Use basic string matching for suggestions
+    const suggestions = findSimilarCommands(command, availableCommands);
+    if (suggestions.length > 0) {
+      console.log('\nDid you mean one of these?');
+      suggestions.forEach((s) => console.log(`- ${s}`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)